source/server.js

```diff
@@ -266,8 +265,9 @@ app.get('/serverdata.js', (req, res) => {
       const text = `ungit.config = ${JSON.stringify(config)};\n` +
         `ungit.userHash = "${hash}";\n` +
         `ungit.version = "${config.ungitDevVersion}";\n` +
-        `ungit.platform = "${os.platform()}"\n` +
-        `ungit.pluginApiVersion = "${require('../package.json').ungitPluginApiVersion}"\n`;
+        `ungit.platform = "${os.platform()}";\n` +
+        `ungit.pluginApiVersion = "${require('../package.json').ungitPluginApiVersion}";\n`;
+      res.set('Content-Type', 'text/javascript');
```

Comment:
All other javascript files have a `Content-Type: application/javascript` header:

![image](https://user-images.githubusercontent.com/4009570/68091893-c58d7b80-fe85-11e9-8b1d-b98bffc7b4aa.png)


```suggestion
      res.set('Content-Type', 'application/javascript');
```